code/controllers/subsystem/economy.dm

```diff
@@ -21,17 +21,28 @@ SUBSYSTEM_DEF(economy)
 	///Multiplied as they go to all department accounts rather than just cargo.
 	var/bounty_modifier = 3
 
+	/// Total value of exported materials.
+	var/export_total = 0
+	/// Total value of imported goods.
+	var/import_total = 0
```

Comment:
i'm almost certain that this garbage is for arconomy inflation and should be deleted